mne_bids_pipeline/scripts/preprocessing/_02_frequency_filter.py

```diff
@@ -68,39 +67,34 @@ def get_input_fnames_frequency_filter(**kwargs):
         path_kwargs['root'] = cfg.deriv_root
         path_kwargs['suffix'] = 'raw'
         path_kwargs['extension'] = '.fif'
+        path_kwargs['processing'] = 'sss'
     else:
         path_kwargs['root'] = cfg.bids_root
+        path_kwargs['suffix'] = None
+        path_kwargs['extension'] = None
+        path_kwmags['processing'] = cfg.proc
     bids_path_in = BIDSPath(**path_kwargs)
 
-    if cfg.use_maxwell_filter:
-        bids_path_in.update(processing="sss")
-
     in_files = dict()
     in_files[f'raw_run-{run}'] = bids_path_in
     _update_for_splits(in_files, f'raw_run-{run}', single=True)
 
-    if (cfg.process_er or config.noise_cov == 'rest') and run == cfg.runs[0]:
-        noise_task = "rest" if config.noise_cov == "rest" else "noise"
-        if cfg.use_maxwell_filter:
-            raw_noise_fname_in = bids_path_in.copy().update(
-                run=None, task=noise_task
-            )
-            in_files["raw_noise"] = raw_noise_fname_in
-            _update_for_splits(in_files, "raw_noise", single=True)
-        else:
-            if config.noise_cov == 'rest':
-                in_files["raw_rest"] = bids_path_in.copy().update(
-                    run=None, task=noise_task)
+    if run == cfg.runs[0]:
+        do = dict(rest=cfg.process_rest, noise=cfg.process_er)
+        for task in ('rest', 'noise'):
+            key = f'raw_{task}'
+            if cfg.use_maxwell_filter:
+                raw_fname = bids_path_in.copy().update(
+                    run=None, task=task)
             else:
-                assert config.noise_cov == 'noise'
-                ref_bids_path = bids_path_in.copy().update(
-                    run=cfg.mf_reference_run,
```

Comment:
I think this logic was wrong before -- `mf_reference_run` shouldn't influence non-maxfiltered data. It should just use `runs[0]` to find the empty-room data, right? If so, the logic here should be correct